packages/commons/src/auth/authData.ts

```diff
@@ -85,9 +85,9 @@ export const UIAuthToken = SharedStandardJWTClaims.merge(
       origin: z.string(),
       value: z.string(),
     }),
-    name: z.string(),
-    family_name: z.string(),
-    email: z.string(),
+    name: z.string().optional(),
+    family_name: z.string().optional(),
+    email: z.string().optional(),
```

Comment:
Is it optional or nullish? Is the `name` field always present and could be null or sometimes not present?

Reply:
It could either be a `string` or `undefined` / not present, in theory, which is what optional should do. I added tests with the token that was causing the error (replacing properties with mocked values), where these 3 properties are not present, and it is working as expected.

However, let's use `nullish`, it's the same but also includes the case where it is set to `null` and not to `undefined`